indico_toolkit/association/positioning.py

```diff
@@ -103,12 +103,68 @@ def get_min_distance(
             min_distance += (page_height * page_difference)
         return min_distance
 
+    @staticmethod
+    def get_vertical_min_distance(
+        above_pos: dict, below_pos: dict, page_height: int = None
+    ) -> float:
+        """
+        Get the vertical minimum distance between two bounding boxes
+        Args:
+            above_pos (dict): the position expected to be above
+            below_pos (dict): to position expected to be below
+            page_height (int, optional): If you want to measure distances across pages, set the OCR page height
+                                         otherwise locations on separate pages will raise an exception.
+                                         Defaults to None.
+
+        Returns:
+            float: minimum distance
+        """
+        add_page_height = False
+        page_difference = abs(above_pos["page_num"] - below_pos["page_num"])
+        if page_difference > 0:
+            if not page_height:
+                raise ToolkitInputError(
+                    "Predictions are not on the same page! Must enter a page height"
+                )
+            else:
+                add_page_height = True
+        min_distance = below_pos["bbTop"] - above_pos["bbBot"]
+
+        if add_page_height:
+            min_distance += (page_height * page_difference)
+        return min_distance
+
+    @staticmethod
+    def get_horizontal_min_distance(
+            left_pos: dict, right_pos: dict) -> float:
```

Comment:
I think black needs to be run based on this formatting

Reply:
also if all you want is the horizontal distance, does it make sense to specify left/right position in the args? i.e. you probably don't have expectations for that yet and/or it's not relevant

Reply:
if you make that change, probably need to change other logic to check for min positive value or something